opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -320,6 +323,8 @@ enum DataPointFlags {
 message NumberDataPoint {
   // The set of key/value pairs that uniquely identify the timeseries from
   // where this point belongs. The list may be empty (may contain 0 elements).
+  // The OpenTelemetry API specification further restricts the allowed value types:
+  // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/common.md#attributes
```

Comment:
Since we have this so many times, I think it would be better to move this comment to the definition of KeyValue.

Reply:
`KeyValue` is also used in `logs.proto` which does not have this restriction (logs were the reason why `KeyValue` was defined this permissively in first place). The comments only apply to `trace.proto` and `metrics.proto` but I agree the duplication is not really nice.